src/main/java/mate/academy/bookstore/model/User.java

```diff
@@ -0,0 +1,32 @@
+package mate.academy.bookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Data;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
+
+@Data
```

Comment:
@ Data on JPA entity is a bad idea, it can cause problems, use only what you need - Getter, @ Setter etc

Reply:
Not fixed